src/nvim/api/keysets.lua

```diff
@@ -21,6 +21,9 @@ return {
     "virt_lines";
     "virt_lines_above";
     "virt_lines_leftcol";
+    "sign_text";
+    "sign_hl_group";
+    "sign_num_hl_group";
```

Comment:
let's just call this `number_hl`. While the functionality was in signs originally, it is just now a decoration changing the number column, without "signs" being involved as a concept (even though the implementation might be entangled for the moment).

Reply:
`number_hl_group` would be more consistent. Otherwise, could we rename `hl_group` -> `line_hl` and keep `hl_group` as an undocumented alias?

Reply:
hmm, let's do `number_hl_group`.